product list/product.css

```diff
@@ -0,0 +1,79 @@
+.product-container{
+    display: flex;
+    margin: 1rem;
+}
+.filter-container{
+    position: fixed;
+    display: flex;
+    flex-direction: column;
+    justify-content: center;
+    gap: .5rem;
+    padding: 0 .5rem;
+    border-right: 1px solid var(--color-border);
+}
+.filter-header{
+    display: flex;
+    justify-content: space-between;
+}
+.center-hr{
+border-top: 1px solid var(--color-border);
+padding: .5rem;
+}
+.header{
+    font-size: 1rem;
+    letter-spacing: 1px;
```

Comment:
Can use css variables for font size and spacings.

Reply:
Yes will try to refactor the code